backend/src/services/tag.service.ts

```diff
@@ -4,30 +4,32 @@ import TagRepository from '../data/repositories/tag.repository';
 import { HttpError } from '../common/errors/http-error';
 import { HttpCode } from '../common/enums/http-code';
 import { HttpErrorMessage } from '../common/enums/http-error-message';
+import { mapTagToITag } from '../common/mappers/tag/map-tag-to-itag';
 
 export const getAllByWorkspaceId = async (
   workspaceId: string,
 ): Promise<ITag[]> => {
-  return getCustomRepository(TagRepository).findAllByWorkspaceId(workspaceId);
+  const tags = await getCustomRepository(TagRepository).findAllByWorkspaceId(workspaceId);
+  return tags.map(mapTagToITag);
 };
 
-export const create = async (newTag: ITagCreation): Promise<ITag> => {
+export const create = async (workspaceId: string, newTag: ITagCreation): Promise<ITag> => {
   const tagRepository = getCustomRepository(TagRepository);
   if (!newTag.name) {
     throw new HttpError({
       status: HttpCode.BAD_REQUEST,
       message: HttpErrorMessage.TAG_EMPTY_STRING,
     });
   }
-  const tagsInDB = await tagRepository.find(newTag);
+  const tagsInDB = await tagRepository.findByNameAndWorkspaceId(workspaceId, newTag.name);
```

Comment:
Will it be more neat?

```suggestion
  const tagsInDB = await tagRepository.find({workspaceId, name: newTag.name});
```